x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.test.ts

```diff
@@ -265,7 +266,15 @@ describe('endpoint list middleware', () => {
           },
         ],
       } as ActivityLog);
-    const dispatchGetActivityLog = () => {
+
+    const dispatchGetActivityLogLoading = () => {
+      dispatch({
+        type: 'endpointDetailsActivityLogChanged',
+        // @ts-ignore
```

Comment:
I'm not opposed to making `ts` chill out in test files. But can we make this more specific? And only ignore the precise thing getting flagged here?

Reply:
@pzl  I think this ignore is ok. Its due to a know issue which we have tech debt to address - issue 830 on our backlog.

@ashokaditya maybe add this above the ignore:

```
// Ignore will be fixed with when AsyncResourceState is refactored (#830)
```

Reply:
done aa63970f202f950c25ce8c91ffa7fddbbd6c0765